src/libraries/System.Text.Json/src/Resources/Strings.resx

```diff
@@ -677,4 +677,25 @@
   <data name="JsonRequiredPropertiesMissing" xml:space="preserve">
     <value>JSON deserialization for type '{0}' was missing required properties, including the following: {1}</value>
   </data>
-</root>
+  <data name="ObjectCreationHandlingAttributeOnWrongJsonTypeInfoKind" xml:space="preserve">
+    <value>JsonObjectCreationHandlingAttribute is not valid on '{0}' because JsonTypeInfoKind for this type is not Object.</value>
+  </data>
+  <data name="ObjectCreationHandlingPopulateNotSupportedByConverter" xml:space="preserve">
+    <value>Property '{0}' on type '{1}' is marked with JsonObjectCreationHandling.Populate but it uses converter which doesn't support populating.</value>
```

Comment:
I think this could be shortened somewhat, e.g.

"Using JsonObjectCreationHandling.Populate is not supported on properties that use custom converters."

Reply:
it's not only custom converters. It's all converters which don't support it. I.e. ImmutableList as well

Reply:
Perhaps then it makes sense to elaborate further in a second sentence, i.e. 

```suggestion
    <value>Property '{0}' on type '{1}' is marked with JsonObjectCreationHandling.Populate but it doesn't support populating. This can be either because the property type is immutable or it could use a custom converter.</value>
```

Reply:
will be included in the next iteration